analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt

```diff
@@ -124,7 +124,7 @@ abstract class AbstractIntegrationSpec : StringSpec() {
                     .resolveDependencies(files)
 
                 results.size shouldBe files.size
-                results.values.forAll { result ->
```

Comment:
commit-message: detekting -> detecting

consider: the actual support for detecting mutliple projects

Reply:
Done.